docs/resources/cluster.md

```diff
@@ -1013,18 +1013,18 @@ The following attributes are exported:
 
 ###### Arguments
 
-* `aep` - (Optional) Attachable entity profile (string)
-* `apic_hosts` - (Optional) List of APIC hosts to connect for APIC API (list)
+* `aep` - (Required) Attachable entity profile (string)
+* `apic_hosts` - (Required) List of APIC hosts to connect for APIC API (list)
 * `apic_refresh_time` - (Optional) APIC refresh time in seconds (string)
```

Comment:
I copied them straight from the Rancher API schema:
https://<rancher-instance>/v3/schemas/aciNetworkProvider
So it's a 1:1 between the Rancher API and the Terraform provider.

I've used the API, and the extra fields (compared to the RKE provider) aren't required for the ACI CNI to function, but I thought it could be useful for others.

Reply:
That isn't working for me. Could you provide another link to the schema so I can check it out?

Reply:
Sorry, it seems markdown removed parts of the URL (I was trying to show a local Rancher instance). I found the schema in the Rancher repo: https://github.com/rancher/rancher/blob/release/v2.6/pkg/client/generated/management/v3/zz_generated_aci_network_provider.go

Reply:
Thank you, that's what I needed